docs/technical/branching-strategy.md

```diff
@@ -0,0 +1,9 @@
+# Branching Strategy
+
+This document outlines the overall branching strategy for all components repositories in the Bürokratt system.
+
+[Trunk Based Development](https://trunkbaseddevelopment.com/) model is used.
+
+# TL;DR
+* Short living feature branches
+* Feature branch goes back to `main` branch after it passes the CI pipeline and code review.
```

Comment:
A few more things that might be worth mentioning:
- Feature branches should only be merge into `main` via PR (might be worth being explicit)
- PRs can only be merged into `main` with squash merge

@turnerrainer mentioned that he wanted to handle releases as branches so having some naming conventions might help manage this. It might be better to suggest that branch names should be prefixed with one the popular conventions:
- `release/*` (these can also be tags)
- `feature/*`
- `hotfix/*`

What do you think?

Reply:
I didn´t follow it myself, will do it from now on.

Reply:
What about rollbacks? If there is a need for it should it be also done via PR? And will it be done through `hotfix` prefix?